doc/source/advanced-use.rst

```diff
@@ -6,246 +6,256 @@ Clones
+This reverses the *clone* and *source* jail relationship. The clone
+becomes the source and the source jail is demoted to a clone.
 
-**Now you can remove the demoted jail with:**
+**The demoted jail can now be removed:**
 
-``iocage destroy UUID | TAG``
+:samp:`# iocage destroy UUID | TAG`
 
 Updating jails
 --------------
 
-Updates are handled with the freebsd-update(8) utility. Jails can be updated
-while they are stopped or running.
+Updates are handled with the freebsd-update(8) utility. Jails can be
+updated while they are stopped or running.
 
-To update a jail to latest patch level run:
+To update a jail to latest patch level, run:
 
-``iocage update UUID | TAG``
+:samp:`# iocage update UUID | TAG`
 
-This will create a back-out snapshot of the jail automatically.
+This automatically creates a back-out snapshot of the jail.
 
-When finished with updating and the jail is working OK, simply remove the snapshot:
+When updates are finished and the jail appears to function properly,
+remove the snapshot:
 
-``iocage snapremove UUID|TAG@snapshotname``
+:samp:`# iocage snapremove UUID|TAG@snapshotname`
 
-In case the update breaks the jail, simply revert back to the snapshot:
+If the update breaks the jail, revert back to the original snapshot:
 
-``iocage rollback UUID|TAG@snapshotname``
+:samp:`# iocage rollback UUID|TAG@snapshotname`
 
-If you'd like to test updating without affecting a jail, create a clone and
-update the clone the same way as outlined above.
+To test updating without affecting a jail, create a clone and update the
+clone the same way as outlined above.
 
-To clone run:
+To clone a jail, run:
 
-``iocage clone UUID|TAG tag=testupdate``
+:samp:`# iocage clone UUID|TAG tag=testupdate`
 
 Upgrading jails
 ---------------
 
-Upgrades are handled with the freebsd-update(8) utility.
-By default the upgrade command will try to upgrade the jail
-to the hosts RELEASE version (uname -r).
+Upgrades are handled with the freebsd-update(8) utility. By default, the
+upgrade command attempts to upgrade the jail to the host's RELEASE
+version (visible with uname -r).
 
-Based on the jail "type" property, upgrades are handled differently
-for basejails and non basejails.
+Based on the jail **type** property, upgrades are handled differently
+for basejails and normal jails.
 
-Upgrade non-basejail
+Upgrade Normal Jails
 ++++++++++++++++++++
 
-To upgrade a normal jail (non basejail) to the hosts RELEASE run:
+To upgrade a normal jail (non-basejail) to the host's RELEASE, run:
 
-``iocage upgrade UUID | TAG``
+:samp:`# iocage upgrade UUID | TAG`
 
-This will upgrade the jail to the same RELEASE as the host.
+This upgrades the jail to the same RELEASE as the host.
 
-To upgrade to a specific release run:
+To upgrade to a specific release, run:
 
-``iocage upgrade UUID|TAG release=10.1-RELEASE``
+:samp:`# iocage upgrade UUID|TAG release=10.1-RELEASE`
 
 Upgrade basejail
 ++++++++++++++++
```

Comment:
The entire basejail section needs a disclaimer that this upgrade mechanism for them is not written yet. So possibly remove it entirely or make it known that this is only for the old legacy implementation for now.